### PR TITLE
[improve][broker] Add `individualDeletedMessagesSize` to log when cusor close.

### PR DESCRIPTION
**Pull request by mattisonchao**
### Motivation

When the topic is unloaded, may cause the redelivery of messages. the cases as below:

- When `individualDeletedMessages` exceeds `maxUnackedRangesToPersist`.

We can add `individualDeletedMessagesSize` to the log to help pinpoint the problem.

### Modifications

- Add `individualDeletedMessagesSize` to log.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Documentation

- [x] `no-need-doc` 
(Please explain why)

### CODE REVIEW

File: managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java [hunk 1]
```diff
@@ -2355,7 +2361,8 @@ public void operationFailed(ManagedLedgerException e) {
             persistPositionMetaStore(-1, position, properties, new MetaStoreCallback<Void>() {
                 @Override
                 public void operationComplete(Void result, Stat stat) {
-                    log.info("[{}][{}] Closed cursor at md-position={}", ledger.getName(), name, markDeletePosition);
+                    log.info("[{}][{}] Closed cursor at md-position={} individualDeletedMessagesSize={}",
+                            ledger.getName(), name, markDeletePosition, individualDeletedMessages.size());
```
Comment (Technoboy-) [inline managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java:2365]: Log `individualDeletedMessages.size()` seems useless.

Comment (mattisonchao) [reply managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java:2365]: Well, it looks like the log here is not accurate, I will add a log in front

File: managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java [hunk 2]
```diff
@@ -2333,6 +2338,9 @@ public void closeFailed(ManagedLedgerException exception, Object ctx) {
      */
     void persistPositionWhenClosing(PositionImpl position, Map<String, Long> properties,
             final AsyncCallbacks.CloseCallback callback, final Object ctx) {
+        log.info("Persistence before cursor close is in progress. position={} "
```
Comment (gaoran10) [inline managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java:2341]: I'm not sure that we need to add this log, it seems that there are logs when this operation was completed successfully or not.

Comment (mattisonchao) [reply managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java]: Move this log to `ManagedCusrorImpl#buildIndividualDeletedMessageRanges `.

**Pull request closed without merging**